### PR TITLE
Included Cg shader support

### PR DESCRIPTION
**Pull request by dbenoy**

### CODE REVIEW

File: org.libretro.RetroArch.json [hunk 1]
```diff
@@ -59,7 +60,30 @@
               "sha256": "0f4620e11fa283fedafb474427c8e96bf149511a1804bdc47350963ae5cf54d8"
             }
           ]
-        }
+        },
+	{
+	  "name": "nvidia-cg-tookit",
```
Comment (RobLoach) [inline org.libretro.RetroArch.json]: This is great! Was wondering why it wasn't working.

Mind switching the whitespace from tabs over to spaces? Thanks!

n7azeef

**Comment by dbenoy** [comment]
Sadly, the particular Cg shader pack I wanted to try is causing segfaults (https://forums.libretro.com/t/analog-shader-pack-version-3-is-here/5493 -- Maybe the segfault has something to do with how I had to change all the backslashes to forward slashes) but I tried other cg shaders and they seem to load.

Glad to have helped with the flatpak build though.

File: org.libretro.RetroArch.json [hunk 2]
```diff
@@ -59,6 +60,29 @@
...
+        {
+          "name": "nvidia-cg-tookit",
+          "sources": [
+          {
+              "type": "archive",
+              "url": "http://developer.download.nvidia.com/cg/Cg_3.1/Cg-3.1_April2012_x86_64.tgz",
+              "sha256": "e8ff01e6cc38d1b3fd56a083f5860737dbd2f319a39037528fb1a74a89ae9878"
+          },
+            {
+              "type": "script",
```
Comment (Barthalion) [inline org.libretro.RetroArch.json]: Please change it to `simple` buildsystem. There is no need to such hackery in flatpak 0.9.0.

Comment (dbenoy) [inline org.libretro.RetroArch.json]: I'm not sure what you're referring to. Is 'simple' some type of flatpak builder syntax I'm not familiar with? Could you point me at some docs?

Or are you suggesting changing the way that the software is built and installed? Believe it or not, just copying the files is how it's done. It's a binary distribution.

Comment (Barthalion) [inline org.libretro.RetroArch.json]: It is described in `flatpak-manifest` man page. I don't think its up-to-date version is online anywhere. You can look at other Flathub packages like [gnucash](https://github.com/flathub/org.gnucash.GnuCash/blob/master/org.gnucash.GnuCash.json#L75) for examples.

Comment (RobLoach) [inline org.libretro.RetroArch.json]: https://github.com/flathub/org.libretro.RetroArch/pull/14

**Pull request merged**